crates/sparrow-session/src/partitioned/partitioned.rs

```diff
@@ -1 +0,0 @@
-mod basic_tests;
```

Comment:
Nothing was referencing `sparrow_session::partitioned::partitioned` which corresponded to this file.